tests/stepDefinitions/loginContext.js

```diff
@@ -1,35 +1,39 @@
 const { Given, When, Then } = require('@cucumber/cucumber')
 const { expect } = require("@playwright/test");
-
-
-// for valid scenario
-Given('user {string} has navigated to the admin login page', function (string) {
-  console.log(" Given ");
+const loginBtnLocator=".btn.btn-block.btn-primary"
```

Comment:
Always give one space before and after '='
```suggestion
const loginBtnLocator = ".btn.btn-block.btn-primary"
```